src/dotnet/APIView/APIView/Model/TreeTokenCodeFile.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using APIView;
+using csharp_api_parser.TreeToken;
+using System;
+using System.Collections.Generic;
+
+namespace ApiView
+{
+    public class TreeTokenCodeFile
+    {
+        private string _versionString;
```

Comment:
Why do we need this `_versionString` and `VersionString`?  `VersionString` can directly used with a custom get method instead of having another private property. right?

Reply:
I have removed TreeTokenCodeFile. Using the existing code file.